src/app/zap-templates/zcl/data-model/chip/unit-localization-cluster.xml

```diff
@@ -35,7 +35,10 @@ limitations under the License.
       mechanism by which they can be configured to use a user’s preferred unit.</description>
     <globalAttribute side="either" code="0xFFFD" value="1"/>
     <!-- Base data types -->
-    <attribute side="server" code="0x00" define="TEMPERATURE_UNIT" type="TempUnitEnum" min="0" max="2" writable="true" optional="true">TemperatureUnit</attribute>
+    <attribute side="server" code="0x00" define="TEMPERATURE_UNIT" type="TempUnitEnum" min="0" max="2" writable="true" optional="true" isNullable="true">
```

Comment:
This is a wire-format and API-compat breaking change.  We need to figure out whether and how we can do this...  In particular, I don't think Apple can ship this change.

Reply:
Updated top  remove the isNullable = true, will have to update spec for that.